Formula/lgeneral.rb

```diff
@@ -0,0 +1,38 @@
+class Lgeneral < Formula
+  desc "Turn-based strategy engine heavily inspired by Panzer General"
+  homepage "https://lgames.sourceforge.io/LGeneral/"
+  url "https://downloads.sourceforge.net/lgeneral/lgeneral/lgeneral-1.4.4.tar.gz"
+  sha256 "0a26b495716cdcab63b49a294ba31649bc0abe74ce0df48276e52f4a6f323a95"
+  license "GPL-2.0-or-later"
+
+  depends_on "gettext"
+  depends_on "sdl12-compat"
+  depends_on "sdl2"
+
+  def install
+    inreplace "configure", "#include <unistd.h>", "#include <sys/stat.h>\n#include <unistd.h>"
+    system "./configure", *std_configure_args,
+                         "--disable-silent-rules",
+                         "--disable-sdltest",
+                         "CFLAGS=-Wl,-rpath,#{Formula["sdl2"].opt_lib}"
```

Comment:
I don't think the rpath should include another formula.

CC @carlocab for guidance here

Reply:
Copied from https://github.com/Homebrew/homebrew-core/blob/master/Formula/sdl12-compat.rb#L52
sdl12 uses sdl2 library.
@carlocab pls commet

Reply:
Setting `RPATH`s this way is fine, but this really should be fixed in `sdl12-compat` rather than in its various users.

Can you try rebuilding `sdl12-compat` with my changes from #102612 to see if it allows you to avoid setting `CFLAGS`?

Reply:
Since that PR is merged, can you remove this now?